lib/pages/video_capture.dart

```diff
@@ -0,0 +1,222 @@
+import 'package:camera/camera.dart';
+import 'package:flutter/material.dart';
+import 'package:gallery_saver/gallery_saver.dart';
+import 'package:path/path.dart';
+import 'package:path_provider/path_provider.dart';
+
+class VideoCapture extends StatefulWidget {
+  static final String route = '/videoCapture';
+
+  VideoCapture();
```

Comment:
No need for default constructor explicitly dart provides one by default. Let's remove this.